web/modules/custom/geolayer/modules/map/src/Plugin/Field/FieldFormatter/GeofieldFormatter.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace Drupal\geolayer_map\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\FormatterBase;
+use Drupal\geofield\GeoPHP\GeoPHPInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+/**
+ * Plugin implementation of the map 'geofield' formatter.
+ *
+ * @FieldFormatter(
+ *   id = "geolayer_map_geofield",
+ *   label = @Translation("Geolayer Map"),
```

Comment:
Since this formatter is form a geofield let's called it "Geofield Map" to distinguish it from the "Geolayer Map" in GeolayerFormatter.php